.travis.yml

```diff
@@ -17,12 +17,10 @@ jdk:
   - oraclejdk8
 
 install:
-  - mvn clean install -DskipTests -Dassembly.skipAssembly=true > /dev/null
+  - bash .travis_install.sh
 
 script:
-  - cd $PINOT_MODULE
-  - cp ../.travis_cmd.sh .
-  - bash .travis_cmd.sh
+  - bash .travis_test.sh
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
```

Comment:
I would agree with JF's comment.

On Thu, Jun 2, 2016 at 11:10 AM, Jean-François Im <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/linkedin/pinot/pull/140#discussion_r65590051>:
>
> >
> >  script:
> > -  - cd $PINOT_MODULE
> > -  - cp ../.travis_cmd.sh .
> > -  - bash .travis_cmd.sh
> > +  - bash .travis_test.sh
> >
> >  after_success:
> >    - bash <(curl -s https://codecov.io/bash)
>
> This could be a security/liability issue if someone compromises the
> codecov.io script to send spam and/or DDoS some other website. It would
> be better to check out a known revision of the script (which is available
> at https://github.com/codecov/codecov-bash) and invokes it as part of a
> script that does the after_success part of the build than fetching an
> unknown version of it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/linkedin/pinot/pull/140/files/0b806a490eba7a6250c08dab0ce795029c2f111e#r65590051>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADSJ7kf9I4qdivmY0dm4kiDM0Gnbqp4pks5qHxyzgaJpZM4IsJ6Y>
> .
>